molecule.yml

```diff
@@ -31,7 +32,7 @@ docker:
     - name: yatm.cont
       ansible_groups:
         - yatm
-      image: geerlingguy/docker-debian9-ansible
+      image: geerlingguy/docker-debian8-ansible
```

Comment:
Maybe both containers, to ensure Debian 8 & 9 compatibility?

Reply:
AFAIK you cannot use a specific container in molecule 1.x.x if you have more than one. 

Could you give me an example?

Reply:
My idea is starting up both containers and test the role against both of them:

``` yaml
docker:
  containers:
    - name: yatm_debian9.cont
      ansible_groups:
        - yatm
      image: geerlingguy/docker-debian9-ansible
      image_version: latest

      privileged: True
      cap_add:
        - SYS_ADMIN
      volume_mounts:
        - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
      command: '/lib/systemd/systemd'

    - name: yatm_debian8.cont
      ansible_groups:
        - yatm
      image: geerlingguy/docker-debian8-ansible
      image_version: latest

      privileged: True
      cap_add:
        - SYS_ADMIN
      volume_mounts:
        - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
      command: '/lib/systemd/systemd'
```

Reply:
good idea! done 👍